task8/testingRestAPI/responseFiles/responseDoPost.json

```diff
@@ -0,0 +1,12 @@
+Post{
```

Comment:
Uh, oh
This is is not compliant with json format, see the github going mad with the text )) 
![image](https://user-images.githubusercontent.com/48795460/117018967-aa5f2e80-acfd-11eb-96fa-18b9d7b55c0f.png)

A JSON entity key must be enclosed in "", and so is a literal value